src/modules/header/components/header.css

```diff
@@ -178,6 +178,9 @@
 .is-open {
   transform: translate(28em, 0);
 }
+.is-active {
+  color: rgba(255,255,255,1) !important;
```

Comment:
the difference to the non-active color is (on my monitor) difficult to perceive.